helm/dagster/schema/schema_tests/test_service_account.py

```diff
@@ -28,6 +29,16 @@ def subchart_helm_template() -> HelmTemplate:
     )
 
 
+@pytest.fixture(name="standalone_subchart_template")
```

Comment:
So this one is interesting. There already was a `subchart_template` fixture, but it's testing in the context of deploying the subchart as part of the primary chart.

I opted to keep both in, and create a `standalone` fixture here for my tests, since the things I'm testing only exist when deploying the subchart as a standalone.